libmscore/undo.cpp

```diff
@@ -1012,7 +1016,8 @@ void ChangeKeySig::flip(EditData*)
 
       showCourtesy = sc;
       ks           = oe;
-      keysig->score()->setLayoutAll();  //TODO: reduce update to range covered by keysig
+      keysig->score()->setLayout(tick);
+      keysig->score()->setLayout(keysig->staff()->nextKeyTick(tick));
```

Comment:
The same can be done in ChangeClefType::flip() to reduce layout to clef range (using Staff::nextClefTick()).

Reply:
Are you going to do that change in `ChangeClefType::flip()` too?